change/@ni-nimble-components-e81dea16-5dd0-4d2d-8814-c5f263d310f1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Was hoping this would fix #801 and allow us to re-enabling `npm audit` for dev dependencies, but I still see 3 critical vulnerabilities in `minimist`. Oh well.